src/commons/eventsourcing/store.rs

```diff
@@ -1,155 +1,69 @@
 use std::{
+    borrow::Cow,
     collections::HashMap,
     fmt,
     str::FromStr,
-    sync::{Arc, RwLock},
+    sync::{Arc, Mutex, RwLock},
 };
 
 use rpki::{ca::idexchange::MyHandle, repository::x509::Time};
-use serde::{de::DeserializeOwned, Deserialize, Serialize};
+use serde::{de::DeserializeOwned, Serialize};
 use url::Url;
 
 use crate::commons::{
-    api::{CommandHistory, CommandHistoryCriteria, CommandHistoryRecord, Label},
+    api::{CommandHistory, CommandHistoryCriteria, CommandHistoryRecord},
     error::KrillIoError,
     eventsourcing::{
-        cmd::{Command, StoredCommandBuilder},
-        locks::HandleLocks,
-        segment, Aggregate, Event, Key, KeyValueError, KeyValueStore, PostSaveEventListener, PreSaveEventListener,
-        Scope, Segment, SegmentBuf, SegmentExt, StoredCommand, WithStorableDetails,
+        cmd::Command, locks::HandleLocks, segment, Aggregate, Key, KeyValueError, KeyValueStore, PostSaveEventListener,
+        PreSaveEventListener, Scope, Segment, SegmentBuf, SegmentExt, StoredCommand, StoredCommandBuilder,
     },
 };
 
+use super::InitCommand;
+
 pub type StoreResult<T> = Result<T, AggregateStoreError>;
 
 //------------ Storable ------------------------------------------------------
 
 pub trait Storable: Clone + Serialize + DeserializeOwned + Sized + 'static {}
 impl<T: Clone + Serialize + DeserializeOwned + Sized + 'static> Storable for T {}
 
```

Comment:
Removing this impacts `06_ca_manager.md`.

Reply:
Created issue: [Revise module structure and update rustdoc #1091](https://github.com/NLnetLabs/krill/issues/1091)